src/main/java/reposense/parser/AuthorConfigCsvParser.java

```diff
@@ -110,6 +110,7 @@ private static void setEmails(AuthorConfiguration config, Author author, List<St
      * Otherwise, use github id from {@code author}.
      */
     private static void setDisplayName(AuthorConfiguration config, Author author, String displayName) {
+        displayName = displayName.replace("\"", "");
```

Comment:
The scope of this is too big. What if a user has a display name of `A"C` ?

Reply:
Made the changes to remove `"` only if detected at the beginning and end of an element.